packages/cfd/src/Containers/__tests__/cfd-change-password-confirmation.spec.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import ChangePasswordConfirmation from '../cfd-change-password-confirmation';
+
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+
+    return {
+        ...original_module,
+        Icon: jest.fn(() => 'mockedIcon'),
+    };
+});
+
+it('should render cfd-change-password-confirmation component', () => {
+    render(<ChangePasswordConfirmation />);
+    expect(screen.getByTestId('dt_cfd_change_password_form')).toBeInTheDocument();
+});
+
+it('should show icon and buttons', () => {
+    render(<ChangePasswordConfirmation />);
+
+    expect(screen.getByText('mockedIcon')).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: 'Cancel' })).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: 'Create' })).toBeInTheDocument();
+});
+
+it('should cancel when cancel button is clicked', () => {
+    render(<ChangePasswordConfirmation platform='mt5' />);
+
+    const el_cancel_btn = screen.getByRole('button', { name: 'Cancel' });
+    fireEvent.click(el_cancel_btn);
+
+    waitFor(() => {
```

Comment:
It is better to wrap this waitFor in await. Refer this documentation https://testing-library.com/docs/guide-disappearance/.
Also since we are testing whether the test is removed, we  can use waitForElementToBeRemoved